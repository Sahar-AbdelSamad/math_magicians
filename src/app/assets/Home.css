.home p {
  font-weight: bold;
  color: #751966;
}
