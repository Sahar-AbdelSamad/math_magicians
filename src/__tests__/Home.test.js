import renderer from 'react-test-renderer';
import Home from '../app/components/Home';

it('renders correctly', () => {
  const tree = renderer
    .create(<Home />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
