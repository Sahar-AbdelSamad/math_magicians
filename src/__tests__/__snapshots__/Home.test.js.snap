// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="home"
>
  <h2>
    Welcome to the homepage!
  </h2>
  <p>
    Mathematics is widely used in science for modeling phenomena. This enables the extraction of quantitative predictions from experimental laws. For example, the movement of planets can be predicted with high accuracy using Newton's law of gravitation combined with mathematical computation. Mathematics is essential in many fields, including natural sciences, engineering, medicine, finance, computer science and social sciences. Some areas of mathematics, such as statistics and game theory, are developed in direct correlation with their applications, and are often grouped under the name of applied mathematics.
  </p>
  <p>
    An electronic calculator is typically a portable electronic device used to perform calculations, ranging from basic arithmetic to complex mathematics. Research suggests that inadequate guidance in the use of calculating tools can restrict the kind of mathematical thinking that students engage in. Others have argued that calculator use can even cause core mathematical skills to atrophy, or that such use can prevent understanding of advanced algebraic concepts.
  </p>
</div>
`;
